File: team15/stuff-supplier/src/app/models/order.ts
import { Address } from "./address";
import { OrderItem } from "./order-item";
import { OrderStatus } from "../enums/order-status";


export class Order {
    id?: number;
    description?: string;
    address?: Address;
    email?: string;
    phoneNumber?: string;
    orderStatus?: number;
    orderItems?: OrderItem[];
}

File: team15/StuffSupplierAPI/StuffSupplierAPI/Model/Order.cs
using StuffSupplierAPI.Model.Enum;

namespace StuffSupplierAPI.Model
{
    public sealed class Order
    {
        public int Id { get; set; }

        public ICollection<OrderItem> OrderItems { get; set; }

        public OrderStatus OrderStatus { get; set; }

        public string Description { get; set; }

        public Address Address { get; set; }

        public string Email { get; set; }

        public string PhoneNumber { get; set; }

    }
}
